analyzers/src/SonarAnalyzer.ShimLayer/ITypeSymbolExtensions.cs

```diff
@@ -3,24 +3,23 @@
 
 namespace StyleCop.Analyzers.Lightup
 {
-
     public static class ITypeSymbolExtensions
     {
         private static readonly Func<ITypeSymbol, bool> IsTupleTypeAccessor;
-        private static readonly Func<ITypeSymbol, bool> IsRefLikeTypeAccessor;
+        private static readonly Func<ITypeSymbol, bool> IsRefLikeTypeAccessor; // Sonar
```

Comment:
In order to keep our changes minimal, I would create a second class for the TypeSymbolExtensions in the Sonar folder and move the new method (and the necessary setup) there.